### PR TITLE
Reword the introduction in the AiMod article

### PR DESCRIPTION
**Pull request by Willexan**
Rewording of AiMod introduction section to improve concision and clarity.
'built it' -> 'built into' 
'Ctrl+Shft+A.'... **once a beatmap is open in...** (specifies premise)
'not detect issues' -> 'overlook issues' (concision)

Reorganize the last paragraph to remove parenthesis for better readability

---

### CODE REVIEW

**Comment by Willexan** [comment]
Hmm how would you rephrase it?
 "...is tool _for modding beatmaps_ built into...?." 
This creates a longer sentence which is less concise don't you think? Ideally the word modding would lead to a dedicated page but I'm not sure if we have one.

**Comment by Krystool** [comment]
I think that "is a tool for beatmap edition" can be okay.

File: wiki/Beatmap_Editor/AiMod/en.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+
+AiMod aims to help beginner mappers "mod" their beatmaps. The limitation of AiMod lies with its basic design which causes it to sometimes overlook issues such as bad mapping patterns, BPM, or timing. Finding a [modder](/wiki/Glossary) to mod your beatmap is strongly recommended.
 
 ![AiMod](AiMod.jpg "AiMod is usually your first modder")
 
-After AiMod checks over your beatmap (should take less than a minute), the warnings and errors are displayed. If "No problems were found in this map!"… Then congratulations! Your beatmap passed the AiMod "modding", but you'll still need to find [modder](/wiki/Glossary) to mod your beatmap because they will find more issues than what AiMod can detect.
+AiMod typically completes its beatmap analysis in under a minute and subsequently displays warnings and errors. If "No problems were found in this map!"… Then congratulations! The beatmap passed the AiMod "modding" but still require modding from a proper modder. The modders can find hidden issues that AiMod failed to detect.
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/AiMod/en.md]: If "No problems were found in this map!"… Then congratulations! -> If no problems were found in your beatmap, then congratulations!

Comment (TPGPL) [reply wiki/Beatmap_Editor/AiMod/en.md]: The beatmap passed the AiMod "modding" but still require modding from a proper modder -> The beatmap passed the AiMod analysis, but it still requires modding from the community.

Comment (Krystool) [reply wiki/Beatmap_Editor/AiMod/en.md]: I approve this correction.

Comment (TPGPL) [reply wiki/Beatmap_Editor/AiMod/en.md]: You don't have to write this every review post, you can support the correction by giving a +1 reaction instead

**Pull request merged**